react/features/invite/functions.ts

```diff
@@ -534,25 +571,37 @@ function isPhoneNumberRegex(): RegExp {
  * @param {string} jwt - The jwt token to pass to the search service.
  * @param {string} text - Text to search.
  * @param {Array<string>} queryTypes - Array with the query types that will be
- * executed - "conferenceRooms" | "user" | "room".
+ * executed - "conferenceRooms" | "user" | "room" | "email".
  * @returns {Promise} - The promise created by the request.
  */
 export function searchDirectory( // eslint-disable-line max-params
         serviceUrl: string,
         jwt: string,
         text: string,
-        queryTypes: Array<string> = [ 'conferenceRooms', 'user', 'room' ]
+        queryTypes: Array<string> = [ 'conferenceRooms', 'user', 'room', 'email' ],
+        peopleSearchTokenLocation?: string,
+        peopleSearchTokenKey?: string,
 ): Promise<Array<{ type: string; }>> {
 
     const query = encodeURIComponent(text);
     const queryTypesString = encodeURIComponent(JSON.stringify(queryTypes));
 
+    // Parse all the query strings of the search directory endpoint
+    const params = new URLSearchParams();
+    params.append('query', query);
+    params.append('queryTypes', queryTypesString);
+    params.append('jwt', jwt);
+
```

Comment:
We should not add jwt as URL param since: https://github.com/jitsi/jitsi-meet/commit/9af0003c639d6ac1b4f2981f44c1e165e88a8a05